src/Mapper/ObjectMapper.php

```diff
@@ -101,12 +93,29 @@ private function map(
             );
         }
 
-        foreach ($this->mappers as $mapper) {
-            if ($mapper->canMap(from: $from, to: $to)) {
-                return $mapper->map(from: $from, to: $to);
+
+        foreach ($this->getMappers() as $mapperClass) {
+            if ($mapperClass->canMap(from: $from, to: $to)) {
+                return $mapperClass->map(from: $from, to: $to);
             }
         }
 
         throw new CannotMapDataException($from, $to);
     }
+
+    /**
+     * @return Mapper[]
+     */
+    protected function getMappers(): array
+    {
+        return [
```

Comment:
One of the issues I've run into with dynamic mappers is that the order sometimes has to be enforced. I don't have good ideas on solving it easily